### ISSUE TITLE
Feature Request: export played cards from the game log file

### ISSUE THREAD

**Issue by AustinYQM** [extesy/DeckTracker#9]
Actually 2, I can split them if that is better for you.

1. If you could allow us to name the decks our opponents played in the tracker that would be great. The tracker lists "Multicolored" but I have a fairly good idea of what they were playing. I would also suggest changing "Multicolored" to "Multicolored (Color 1, Color 2, Color 3)" or something of that nature since mixing all hard to identify three color decks together isn't super helpful statistically.

2. The ability to click on an opponents deck and see what cards they played. Being able to see I lost nearly every-time someone played Steward of the Past is more useful then knowing I lost to feln control. The ability to export the data as a CSV for importing into spread sheets may also be helpful.

**Comment by extesy** [comment]
Hi @AustinYQM, thanks for suggestions. Your first request could be covered by editing [deck type definition file](https://github.com/extesy/DeckTracker/blob/master/decktypes.txt). Adding subtypes for multi-color decks should be very easy, but assigning deck archetypes is less straightforward and I would appreciate any help. See existing examples for Big Combrei, Stonescar Burn, etc.

As for your second suggestion, all historical details for the games you played are already available in `C:\Users{YourProfile}\AppData\Local\UniversalDeckTracker\games.log` file. Although it's not supposed to be readable by humans, but the data is there. Since winrates only make sense in aggregate, seeing/exporting cards for the opponent deck means all games that match that deck archetype, not just a single game. It would probably make more sense to have a separate "game log" screen with a flat list of games you played so that you could copy it from there.

**Comment by AustinYQM** [comment]
Noted, I just made another ticket with some archetype suggestions. I am going to fork the project and see if I can't add some myself and submit a pull request.

**Comment by AustinYQM** [comment]
Also on my first request, if users were allowed the make custom names for decks, perhaps from the overlay while playing (maybe r-click on the overlay for their opponent and select "Custom Achetype"), and the tracker was made to realize a custom deck name was put in then adding some sort of data collection for that specific deck could do wonders for auto generating (or as close as one can get) archetype names.

If the tracker could see the player named a deck "Burn Queen" it could then save the cards played in that deck, as well as the custom name, and send that information back. Compiling that information as seeing the trends could make it much easier to identify decks in the future.

**Comment by AustinYQM** [comment]
Quick question, in decktypes.txt
`||$FJP|Colors.Contains("Fire","Justice","Primal")`
What is the $ for on the deck name? I only see it used on categories (Factions, Mono) else where. If I am adding all the different 3 color combinations should I also make them categories? Should FJP not be a category and function more along the lines of some of the other color combinations that have sub combinations? like Rankano:

```
||Rakano|Colors.Contains("Fire","Justice")
|||Traditional Armory|Cards.ContainsAny(3,"Rakano Artisan","Rolant's Favor","Armorsmith","Sword of Icaria")
```

Just want to make sure before I mess something up.

**Comment by extesy** [comment]
`$` in the beginning of the name means not expose this deck as an archetype and instead just use it for grouping purposes. System will choose the deepest level with a single matching archetype. Since FJP is on the same level as 2-color archetypes, it will match another three and it won't be selected. You can work around that by changing line 9 to `Colors.Count==2` (strict comparison), but then it will ignore decks with only 1 or 2 of off-color cards.

You should probably wait until I add ability to load your custom `decktypes.txt` file in #10 so that you can experiment locally.

**Comment by extesy** [comment]
Thanks for the PR, by the way! It looks good to me on the first glance. I'll test it later today.

**Comment by AustinYQM** [comment]
I figured that was what the `$` was for, or something similar. I changed `$FJP` in the pull request. Yeah how to handle decks that "splash" a third color is a tricky proposition. I noticed that almost every armory deck I was playing against (Which tend to splash Shadow now for Smugglers and Quarry) gets thrown into multicolor. Hopefully the PR I submitted will fix that. 

Let me know if I did anything wrong, once I know I am on the right track I will continue to fill out some more decks.
